### PR TITLE
Change browse to hub

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Angor/Client/Pages/View.razor [hunk 1]
```diff
@@ -591,8 +616,11 @@
             {
                 var words = await passwordComponent.GetWalletAsync();
                 var nostrKey = _derivationOperations.DeriveProjectNostrPrivateKey(words, founderProject.ProjectIndex);
-                var nsec = NBitcoin.DataEncoders.Encoders.Hex.EncodeData(nostrKey.ToBytes());
-                NostrSecKey = nsec;
+                var hex = NBitcoin.DataEncoders.Encoders.Hex.EncodeData(nostrKey.ToBytes());
```
[the remark this answers is not in the document]
Comment (miladsoft) [reply src/Angor/Client/Pages/View.razor:619]: It was already there. I just added nsec

But I will fix it now. It is extra.

File: src/Angor/Client/Pages/View.razor [hunk 2]
```diff
@@ -394,10 +416,15 @@
     string myProjectExplorerLink;
     string projectExplorerLink;
 
-    private string NostrSecKey { get; set; } = string.Empty;
+    private string NostrNsecSecKey { get; set; } = string.Empty;
+    private string NostrHexSecKey { get; set; } = string.Empty;
+
+    private string Npub => NostrConversionHelper.ConvertHexToNpub(project.ProjectInfo.NostrPubKey)!;
```
Comment (dangershony) [inline src/Angor/Client/Pages/View.razor:422]: is this object always guaranteed to be not null?

File: src/Angor/Shared/Utilities/NostrConversionHelper.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        public static string? ConvertBech32ToHex(string bech32Key)
+        {
+            try
+            {
+                string? hrp;
+                return NostrConverter.ToHex(bech32Key, out hrp);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error converting Bech32 to hex: {ex.Message}");
```
Comment (dangershony) [inline src/Angor/Shared/Utilities/NostrConversionHelper.cs:20]: We don't use console we use a logger class, as this is a static class so just throw the exception up

File: src/Angor/Shared/Utilities/NostrConversionHelper.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
+﻿using Nostr.Client.Utils;
```
[the remark this answers is not in the document]
Comment (miladsoft) [reply src/Angor/Shared/Utilities/NostrConversionHelper.cs:1]: it is on the Nostr.Client package

File: src/Angor/Client/Pages/View.razor [hunk 5]
```diff
@@ -591,8 +618,9 @@
             {
                 var words = await passwordComponent.GetWalletAsync();
                 var nostrKey = _derivationOperations.DeriveProjectNostrPrivateKey(words, founderProject.ProjectIndex);
-                var nsec = NBitcoin.DataEncoders.Encoders.Hex.EncodeData(nostrKey.ToBytes());
-                NostrSecKey = nsec;
+                NostrHexSecKey = NBitcoin.DataEncoders.Encoders.Hex.EncodeData(nostrKey.ToBytes());
+                NostrNsecSecKey = NostrConversionHelper.ConvertHexToNsec(NostrHexSecKey)!;
+
```
Comment (dangershony) [inline src/Angor/Client/Pages/View.razor:623]: I think there should be a catch also for regular exception and logs

File: src/Angor/Shared/Utilities/NostrConversionHelper.cs (hunk 3 above)
Comment (dangershony) [reply src/Angor/Shared/Utilities/NostrConversionHelper.cs:20]: ok fine with me

File: src/Angor/Client/Pages/View.razor (hunk 2 above)
Comment (miladsoft) [reply src/Angor/Client/Pages/View.razor:422]: removed !

Comment (miladsoft) [reply src/Angor/Client/Pages/View.razor]: because I used another var in the html codes

**Review by dangershony — APPROVED** [review]

**Pull request merged**